conda/gateways/disk/create.py

```diff
@@ -178,6 +178,28 @@ def create_hard_link_or_copy(src, dst):
         shutil.copy2(src, dst)
 
 
+def _do_softlink(src, dst):
+    if is_executable(src):
+        # for extra details, see https://github.com/conda/conda/pull/4625#issuecomment-280696371
+        # We only need to do this copy for executables which have an RPATH containing $ORIGIN
+        #   on Linux, so `is_executable()` is currently overly aggressive.
+        # A future optimization will be to copy code from @mingwandroid's virtualenv patch.
```

Comment:
Not really, I'm a big fan of detailed commit messages, and would rather you added my commentary to the commit message. Think of the developers on air-planes (ones without internet).